manticore/platforms/linux.py

```diff
@@ -1228,43 +1244,40 @@ def _open(self, f) -> int:
             self.files.append(f)
         return fd
 
-    def _close(self, fd: int) -> None:
+    def _close(self, fd):
         """
         Removes a file descriptor from the file descriptor list
         :rtype: int
         :param fd: the file descriptor to close.
-        :return: C{0} on success.
         """
         try:
-            self.files[fd].close()
-            self._closed_files.append(
-                self.files[fd]
-            )  # Keep track for SymbolicFile testcase generation
+            f = self.files[fd]
+            if f is None:
+                raise FdError(f"Bad file descriptor ({fd})", errno.EBADF)
```

Comment:
This error-handling case was missed previously, and we don't seem to have tests that exercise it.

Reply:
Good catch!